ocaml/xapi/xapi_network.ml

```diff
@@ -20,23 +20,26 @@ open D
 open Db_filter
 open Network
 
-let create_internal_bridge ~__context ~bridge ~uuid =
+let create_internal_bridge ~__context ~bridge ~uuid ~persist =
 	debug "Creating internal bridge %s (uuid:%s)" bridge uuid;
 	let dbg = Context.string_of_task __context in
 	let current = Net.Bridge.get_all dbg () in
 	if not(List.mem bridge current) then
 		let other_config = ["network-uuids", uuid] in
-		Net.Bridge.create dbg ~name:bridge ~other_config ()
+		Net.Bridge.create dbg ~name:bridge ~other_config ();
+	Net.Bridge.set_persistent dbg ~name:bridge ~value:persist
```

Comment:
The indentation suggests that you intended this statement to be outside the `then` clause of the `if` statement above (i.e. always execute it). This is not the case, however: the `let` binding on line 28 sucks all the semicolon-separated statements below it into the `then` clause.

In a top-level, try this:
```
if false then let () = () in print_endline "A"; print_endline "B";;
```
vs.
```
if false then print_endline "A"; print_endline "B";;
```

Or perhaps you knew this and the indentation is wrong?